src/Filesystem/Folder.php

```diff
@@ -345,7 +345,7 @@ public static function isAbsolute($path)
      */
     public static function isRegisteredStreamWrapper($path)
     {
-        return preg_match('/^[^:\/\/]+?(?=:\/\/)/i', $path, $matches) &&
+        return preg_match('/^[^:\/\/]+?(?=:\/\/)/', $path, $matches) &&
```

Comment:
Can this be a functional regression? isAbsolute() still uses /i for example.

Reply:
Nope, the regex has no any characters. `i` is not needed.